cmake/BLTPrivateMacros.cmake

```diff
@@ -8,6 +8,52 @@ include(CMakeParseArguments)
 ## Internal BLT CMake Macros
 
 
+##-----------------------------------------------------------------------------
+## blt_determine_scope(TARGET <target>
+##                     SCOPE  <public (Default)| interface | private >
+##                     OUT    <out variable name>)
+##
+## Returns the scope from a given target and scope string
+##-----------------------------------------------------------------------------
+macro(blt_determine_scope)
```

Comment:
I am a little confused on why the `SCOPE` argument is needed here, but, I am probably missing something.... 

It seems that the intent of the `blt_determine_scope()` macro is to determine the scope of a "target", i.e., PUBLIC, INTERFACE, etc.  Why does the caller need to specify a `SCOPE` argument? Could this macro just take two arguments: (a) the target, and (b) `SCOPE` as an output argument? 

Please, correct me if I misunderstood the intent of this.